src/main/java/com/swervedrivespecialties/swervelib/rev/NeoSteerControllerFactoryBuilder.java

```diff
@@ -98,7 +98,7 @@ public ControllerImplementation create(NeoSteerConfiguration<T> steerConfigurati
     }
 
     public static class ControllerImplementation implements SteerController {
-        private static final int ENCODER_RESET_ITERATIONS = 500;
+        private static final int ENCODER_RESET_ITERATIONS = 100;
```

Comment:
Probably should remove unused variables like this, or at least comment them out.

Reply:
I'll comment them out for now in case we *absolutely* need them- I'm guessing we won't lol